tests/device_tests/tx_cache.py

```diff
@@ -54,7 +60,7 @@ def get_tx(self, txhash):
         # cache miss, try to use backend
         data = self.fetch(txhash)
         with open(cache_file, "w") as f:
-            json.dump(data, f)
+            json.dump(data, f, default=decimal_default)
```

Comment:
Why do you need these changes?

Reply:
It doesn't seem to be necessary, I have removed it.